reporter/src/main/kotlin/reporters/opossum/OpossumReporter.kt

```diff
@@ -567,3 +571,8 @@ class OpossumReporter : Reporter {
         return listOf(outputFile)
     }
 }
+
+private fun DependencyNode.getDependencies(): List<DependencyNode> =
+    buildList {
+        visitDependencies { this += it }
```

Comment:
I am not sure whether this works in all constellations, especially for dependency graph nodes. Here `DependencyNode` is implemented by the [DependencyRefCursor](https://github.com/oss-review-toolkit/ort/blob/babdd1243a283513ccd1590794a10d2afdcdeb9b/model/src/main/kotlin/DependencyGraphNavigator.kt#L125) class. For efficiency reasons, this class iterates over dependencies by changing its own attributes and returning itself. So the list of dependency nodes will only contain a single instance. You will probably have to call `getStableReference()` to get different instances.

Reply:
Thanks, nice catch! Fix it.